tests/e2e/specs/keplr/keplr-spec.js

```diff
@@ -76,4 +77,10 @@ describe('Keplr', () => {
       });
     });
   });
+
+  it(`should get wallet address`, () => {
+    cy.getWalletAddress().then(walletAddress => {
+      expect(walletAddress.length).to.be.equal(45);
+    });
```

Comment:
At the moment, validating it based on the length. Because cosmos address seems to change in subsequent iterations of running tests. But once I add select chain implementation in our `importWallet` flow, we will check the exact value over here.

Reply:
why not get the agoriclocal address instead? that is always constant

Reply:
Yeah, that chain still has to be selected for its address to be visible when collecting wallet addresses. For the implementation I'm referring to, I am planning to select agoric as well as agoric-loca in itl.